scheduling-api-http/build.gradle

```diff
@@ -57,11 +57,23 @@ dependencies {
     // in order to start the application from your IDE
     providedRuntime 'org.springframework.boot:spring-boot-starter-tomcat'
 
-    compile 'org.hibernate:hibernate-entitymanager'
-    compile 'org.hibernate:hibernate-core'
+    compile 'org.hibernate:hibernate-entitymanager:5.1.16.Final'
+    compile 'org.hibernate:hibernate-core:5.1.16.Final'
 
     runtime 'org.hsqldb:hsqldb'
 
+    //support for Oracle DB version 12.1.0.2
+    providedCompile group: 'com.oracle', name: 'ojdbc7', version: '12.1.0.2'
```

Comment:
why not testCompile ?

oracle will not be provided when we compile this project in the release build

Reply:
changed to 'testCompile' and commented because Oracle jdbc dependency can be downloaded only with credentials. I just kept it there to mention the version used to test Oracle DB for future needs.